### PR TITLE
Remove ruby sass compilation

### PR DESCRIPTION
**Pull request by efischer19**
Inspired by my desire to stop getting errors when re-installing ora on a fresh devstack. After ruby was removed from edx-platform, I was getting errors on `gem install sass` in `Makefile`, so I decided to try and copy the work done in https://github.com/edx/edx-platform/commit/c13e62306760b892e3b257fe042dc5b27a9c1d4b in this repo.

I've attempted to keep all the functionality the same, and only had a make a few minor changes (`s/Calc/calc`) to get compilation to succeed. Feel free to correct anything I may have done improperly!

@andy-armstrong @ormsbee Could you review this when you have a minute? No rush at all.

FYI @dianakhuang @robrap @benpatterson

### CODE REVIEW

**Comment by edx-webhook** [comment]
## PR Cover Letter
- [ ] A detailed description of the changes in the body of the PR
- [ ] If available, a link to the relevant JIRA ticket in the body of the PR
  - `[PROJ-XXXX](https://openedx.atlassian.net/browse/PROJ-XXXX)`

## Sandbox
- [ ] Build a sandbox for your branch and add a link here

## Testing
- [ ] i18n
- [ ] RTL
- [ ] Unit, integration, acceptance tests as appropriate
- [ ] Analytics
- [ ] Performance
- [ ] Database migrations are backwards-compatible

## Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [ ] Code review #1
- [ ] Code review #2
- [ ] Docs review (required for UI strings/error messages)
- [ ] UX review
- [ ] Accessibility review
- [ ] Product review

## Documentation considerations
- [ ] Are there user visible changes? If so, list who is affected, and include a pointer to the relevant code.
- [ ] Should this be mentioned in the release notes? If so, make sure that you included a description.

## DevOps assistance
If it's needed, or even if you suspect it might be needed, do any/all these things as early as possible. Don't surprise devops right before merging! For example, touching envs/aws.py or or envs/common.py are dead giveaways that you'll need assistance when pushing your changes out. Things that require devops attention include, but are not limited to:
- configuration changes
- feature flags
- migrations
Ways to get devops attention, pick-and-choose whichever is most appropriate for your specific PR:
- [ ] Tag devops on your PR (`@edx/devops`), either for review (more specific to devops = better) or just a general FYI
- [ ] File a support ticket

FYI: Tag anyone who might be interested in this PR here.

## Post-review
- [ ] Squash commits

File: Makefile [hunk 1]
```diff
@@ -45,7 +45,6 @@ install-sys-requirements: install-system install-node
 	npm config set loglevel warn
 
 install-dev:
-	gem install sass
```
Comment (efischer19) [inline Makefile]: This was the line that kept erroring out on fresh devstacks - it's included as part of `make install`, and `gem` is no longer installed on the box by default.

**Comment by ormsbee** [comment]
:+1:

**Comment by andy-armstrong** [comment]
:+1: Thanks for tackling this!

**Comment by efischer19** [comment]
@andy-armstrong, after this change, on new sass changes to ora2, I end up with `css.map` files that I don't recognize. Should these be added to the edx-ora2 repo? Did I miss a flag to the libsass compiler?

Example: https://github.com/edx/edx-ora2/commits/efischer/separate_commits. I made 2 separate sass changes, then compiled them together, and ended up with 4 new files, not 2.

**Comment by andy-armstrong** [comment]
@efischer19 These are source map files which are useful for debugging. They should not be checked in, so I recommend adding them to the .gitignore file:

https://github.com/edx/edx-platform/blob/master/.gitignore#L79

For some background, here's a good article on source maps: http://blog.teamtreehouse.com/introduction-source-maps

**Comment by efischer19** [comment]
Excellent, thanks for the link!

**Pull request merged**